packages/discord-utilities/src/lib/limits.ts

```diff
@@ -157,5 +193,10 @@ export const UserLimits = {
 	/**
 	 * Maximum numbers of users in a DM group.
 	 */
-	MaximumUsersPerDMGroup: 10
+	MaximumUsersPerDMGroup: 10,
+
+	/**
+	 * Maximum characters allowed in a user's biography (the "About Me" section).
+	 */
+	MaximumBiographyLength: 190
```

Comment:
We can keep it but this isn't something that we can alter via api calls as bot users 😛

Reply:
Yet. Pretty sure Ian said in December's stage event that bots may get biographies in the future.